components/spi/.tekton/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+resources:
```

Comment:
not included in `components/spi/kustomization.yaml` to be applied.

Reply:
@Michkov Just to confirm, what you are saying is I should add `.tekton/` to the `components/spi/kustomization.yaml`,

Reply:
yes, otherwise it will not be applied.

Reply:
Added, PTAL.

Reply:
Also, please let me know if there are any other changes I should make or if this is good to be merged?